jgi_assembly.wdl

```diff
@@ -12,7 +12,7 @@ workflow jgi_metaAssembly{
         String? memory
         String? threads
         # longReads parameters
-        Array[File] input_files
+        Array[String] input_files
```

Comment:
Why did this change to a string? We need this to be of type File for the data staging to work correctly if we want to run on Tahoma.